.NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs

```diff
@@ -68,9 +68,10 @@ protected override void InitializeConfiguration()
             RegisterModel<DateTimeModel>(
                 Culture.Chinese,
                 options => new DateTimeModel(
-                    new FullDateTimeParser(
```

Comment:
FullDateTimeParser was not modified in this PR? What the key difference between it and the new BaseCJKMergedDateTimeParser?

Reply:
They should be the same, I introduced BaseCJKMergedDateTimeParser instead of keep using FullDateTimeParser to be in line with the standard implementation. 
I think FullDateTimeParser can be removed once the refactoring of Japanese and Korean is completed.